src/main/java/Main.java

```diff
@@ -0,0 +1,7 @@
+import controller.LadderController;
+
+public class Main {
+    public static void main(String[] args) {
+        new LadderController().run();
```

Comment:
```suggestion
        new LadderController(). gameStart();
```
은 어떨까요? run() 메소드가 필요할까요?